### PR TITLE
Support tools Unit tests

### PR DESCRIPTION
**Pull request by DawoudSheraz**
### Description
Adds unit tests for:

- Course Summary
- User Summary

### CODE REVIEW

File: src/users/CourseSummary.test.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    expect(componentPropData.key).toEqual(expectedPropData.key);
+    expect(componentPropData.uuid).toEqual(expectedPropData.uuid);
+    expect(componentPropData.title).toEqual(expectedPropData.title);
+    expect(componentPropData.level).toEqual(expectedPropData.level);
+    expect(componentPropData.marketingUrl).toEqual(expectedPropData.marketingUrl);
+    expect(componentPropData.courseRuns).toMatchObject(expectedPropData.courseRuns);
+  });
+
+  it('Missing Course Run Information', () => {
+    const courseData = { ...CourseSummaryData.courseData, courseRuns: [] };
```
Comment (awaisdar001) [inline src/users/CourseSummary.test.jsx:27]: 👍

File: src/users/CourseSummary.test.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import { mount } from 'enzyme';
+import React from 'react';
+
+import CourseSummary from './CourseSummary';
+import CourseSummaryData from './data/test/coursesummary';
+
+describe('Course Summary', () => {
```
Comment (awaisdar001) [inline src/users/CourseSummary.test.jsx:7]: can we have a snapshot test for this component?

**Review by awaisdar001 — APPROVED** [review]
just a nit but rest looks good.

Please change  `/usersummary.js` => `userSummary.js`

**Comment by awaisdar001** [comment]
Good to see `CodeCov +10.32%`

File: src/users/CourseSummary.test.jsx (hunk 2 above)
Comment (DawoudSheraz) [reply src/users/CourseSummary.test.jsx:7]: I am not sure of the exact utility we will be getting with the snapshot tests

**Comment by awaisdar001** [comment]
@DawoudSheraz There are little edge cases still not covered by these thests. please take a look.
https://app.codecov.io/gh/edx/frontend-app-support-tools/compare/94/changes

**Comment by DawoudSheraz** [comment]
> @DawoudSheraz There are little edge cases still not covered by these thests. please take a look.
> https://app.codecov.io/gh/edx/frontend-app-support-tools/compare/94/changes

I saw the coverage on the local. Very basic things uncovered(like closing modal and verifying it has closed).

**Review by azanbinzahid — APPROVED** [review]

File: src/users/CourseSummary.test.jsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  });
+
+  it('Missing Course Run Information', () => {
+    const courseData = { ...CourseSummaryData.courseData, courseRuns: [] };
+    const summaryData = { ...CourseSummaryData, courseData };
+    wrapper = mount(<CourseSummary {...CourseSummaryData} courseData={summaryData} />);
+    expect(wrapper.html()).toEqual(expect.stringContaining('No Course Runs available'));
+  });
+
+  it('Render loading page if data is not present', () => {
```
Comment (Ali-D-Akbar) [inline src/users/CourseSummary.test.jsx:33]: Just a nit (optional), the description for each tests should be Indefinite Tense. There are many instances like this.
Render -> Renders
Hide -> Hides
